client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +229,288 @@ def process(self, instance):
+                "verbose",
+                label="Verbose",
+                default=cls.verbose,
+                tooltip="Prints the current frame that is being evaluated."
+            ),
+            "visibleOnly": BoolDef(
+                "visibleOnly",
+                label="Visible Only",
+                default=cls.visibleOnly,
+                tooltip="Only export dag objects visible during frame range."
+            ),
+            "wholeFrameGeo": BoolDef(
+                "wholeFrameGeo",
+                label="Whole Frame Geo",
+                default=cls.wholeFrameGeo,
+                tooltip=(
+                    "Data for geometry will only be written out on whole "
+                    "frames."
+                )
+            ),
+            "worldSpace": BoolDef(
+                "worldSpace",
+                label="World Space",
+                default=cls.worldSpace,
+                tooltip="Any root nodes will be stored in world space."
+            ),
+            "writeColorSets": BoolDef(
+                "writeColorSets",
+                label="Write Color Sets",
+                default=cls.writeColorSets,
+                tooltip="Write vertex colors with the geometry."
+            ),
+            "writeFaceSets": BoolDef(
+                "writeFaceSets",
+                label="Write Face Sets",
+                default=cls.writeFaceSets,
+                tooltip="Write face sets with the geometry."
+            ),
+            "writeNormals": BoolDef(
+                "writeNormals",
+                label="Write Normals",
+                default=cls.writeNormals,
+                tooltip="Write normals with the deforming geometry."
+            ),
+            "writeUVSets": BoolDef(
+                "writeUVSets",
+                label="Write UV Sets",
+                default=cls.writeUVSets,
+                tooltip=(
+                    "Write all uv sets on MFnMeshes as vector 2 indexed "
+                    "geometry parameters with face varying scope."
+                )
+            ),
+            "writeVisibility": BoolDef(
+                "writeVisibility",
+                label="Write Visibility",
+                default=cls.writeVisibility,
+                tooltip=(
+                    "Visibility state will be stored in the Alembic file. "
+                    "Otherwise everything written out is treated as visible."
+                )
+            )
+        }
+
+        defs = super(ExtractAlembic, cls).get_attribute_defs()
+
+        defs.extend([
+            UISeparatorDef("sep_alembic_options"),
+            UILabelDef("Alembic Options"),
+        ])
+
+        # The Arguments that can be modified by the Publisher
+        overrides = set(cls.overrides)
+        for key, value in override_defs.items():
+            if key not in overrides:
+                continue
+
+            defs.append(value)
+
+        defs.append(
+            UISeparatorDef("sep_alembic_options")
```

Comment:
Should we include a check whether any overrides were enabled to begin with - otherwise one might see an "empty alembic export options" section?

Reply:
done